compiler/src/iree/compiler/Codegen/Common/StripConfigInfoPass.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Common/Passes.h"
+
+#define DEBUG_TYPE "iree-codegen-strip-config-info"
+
+namespace mlir::iree_compiler {
+
+#define GEN_PASS_DEF_STRIPCONFIGINFOPASS
+#include "iree/compiler/Codegen/Common/Passes.h.inc"
+
+namespace {
+class StripConfigInfoPass final
+    : public impl::StripConfigInfoPassBase<StripConfigInfoPass> {
+  using impl::StripConfigInfoPassBase<
+      StripConfigInfoPass>::StripConfigInfoPassBase;
+
+public:
+  void runOnOperation() override;
+};
+} // namespace
+
+void StripConfigInfoPass::runOnOperation() {
+  auto funcOp = getOperation();
+  IREE::Codegen::TranslationInfoAttr translationInfo =
+      getTranslationInfo(funcOp);
+  if (translationInfo) {
```

Comment:
We don't have to keep `translationInfo` as a local variable -- we never use it beyond checking that it's there. I'd do the same thing as you do with lowering config below.